src/renderer/widgets/WalletDetails/model/wc-details-model.ts

```diff
@@ -1,26 +1,45 @@
-import { createEvent, createStore, forward, sample } from 'effector';
+import { createEffect, createEvent, createStore, forward, sample } from 'effector';
 import { combineEvents, spread } from 'patronum';
 
-import { walletModel } from '@entities/wallet';
+import { accountUtils, walletModel } from '@entities/wallet';
 import { walletConnectModel, InitConnectParams } from '@entities/walletConnect';
 import { ReconnectStep, ForgetStep } from '../lib/constants';
 import { walletProviderModel } from './wallet-provider-model';
 import { walletSelectModel } from '@features/wallets';
 import type { Wallet, WalletConnectAccount } from '@shared/core';
 import { chainsService } from '@entities/network';
 import { toAccountId } from '@shared/lib/utils';
+import { AccountId } from '@shared/core';
+import { useBalanceService } from '@entities/balance';
+
+const balanceService = useBalanceService();
 
 const reset = createEvent();
 const confirmReconnectShown = createEvent();
 const reconnectStarted = createEvent<Omit<InitConnectParams, 'client'> & { currentSession: string }>();
 const reconnectAborted = createEvent();
 const sessionTopicUpdated = createEvent();
-const forgetButtonClicked = createEvent();
+const forgetButtonClicked = createEvent<Wallet>();
 const forgetModalClosed = createEvent();
 
 const $reconnectStep = createStore<ReconnectStep>(ReconnectStep.NOT_STARTED).reset(reset);
 const $forgetStep = createStore<ForgetStep>(ForgetStep.NOT_STARTED).reset(reset);
 
+const deleteWalletBalancesFx = createEffect(async (accountsIds: AccountId[]): Promise<void> => {
+  try {
+    await balanceService.deleteBalances(accountsIds);
```

Comment:
I am not sure that it should be here, Why not in balance-model? 

@tuul-wq what do you think?